src/utilities/new_versions.jl

```diff
@@ -12,6 +12,10 @@ title_parenthetical(::KeepEntry) = " (keep existing compat)"
 title_parenthetical(::DropEntry) = " (drop existing compat)"
 title_parenthetical(::NewEntry) = ""
 
+function new_compat_entry(::EntryType, old_compat::Nothing, new_compat::AbstractString)
+    return "$(strip(new_compat))"
+end
+
```

Comment:
since we both added these in different lines of the file this is a duplicate and can be removed.